acceptance/Rakefile

```diff
@@ -90,7 +93,17 @@ EOS
   tests = ENV['TESTS'] || ENV['TEST']
   tests_opt = "--tests=#{tests}" if tests
 
-  config_opt = "--hosts=#{config}" if config
+  target = ENV['TEST_TARGET']
+  if target
+    cli = BeakerHostGenerator::CLI.new([target, '--disable-default-role'])
+    ENV['CONFIG'] = "tmp/#{target}-#{SecureRandom.uuid}.yaml"
+    File.open(config, 'w') do |fh|
```

Comment:
I think @MikaelSmith mentioned in one of these PRs that we need to `Dir.mkdir('tmp')` for this file operation to succeed.